Applications/FileIO/TetGenInterface.cpp

```diff
@@ -492,8 +492,9 @@ bool TetGenInterface::parseElements(std::ifstream& ins,
         }
         // insert new element into vector
         auto** tet_nodes = new MeshLib::Node*[4];
-        for (unsigned k(0); k<4; k++) {
-            tet_nodes[k] = nodes[ids[k]];
+        for (int n = 0; n < 4; n++)
```

Comment:
If the tet is of quadratic order is looping only over the four nodes still okay?

Reply:
I guess there is no test for this.
Concerning the for loop change, it is consistent though with the tet_nodes size.